scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -31,6 +33,13 @@ import java.util.concurrent.TimeUnit
 fun main(vararg args: String) {
   val repoRoot = args[0]
   val filePath = args[1]
+  val reportFormat = when (args.getOrNull(2)) {
```

Comment:
I suggest instead taking as an argument the path to the report file to generate and then verify that it ends in either ".md" or ".html" (and error otherwise). The extension then acts as your mode, and you don't need to default any assumptions on file location or name.

Reply:
This is also addressed as per https://github.com/oppia/oppia-android/pull/5443#issuecomment-2192915627